ga4gh/datamodel/__init__.py

```diff
@@ -187,14 +187,23 @@ def obfuscate(cls, idStr):
         fashion. This is not intended for security purposes, but rather to
         dissuade users from depending on our internal ID structures.
         """
-        return base64.b64encode(idStr)
+        return base64.b64encode(idStr).replace(b'=', b'')
```

Comment:
We should use ``base64.urlsafe_b64encode``, as these are still not URL safe. 

Thanks for the pointer @diekhans.